Dockerfile

```diff
@@ -6,9 +6,16 @@ RUN apt-get update \
  && apt-get install -y --no-install-recommends \
         build-essential \
         clang \
-        cmake \
         python \
         git \
         ninja-build \
+        curl \
+        ca-certificates \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
+
+# Need a more recent cmake than ships with stretch
+RUN curl -sSfL \
+    https://github.com/Kitware/CMake/releases/download/v3.17.0/cmake-3.17.0-Linux-x86_64.tar.gz \
+    | tar xzf -
```

Comment:
Looks like that solution is to use the `-S` and `-B` together maybe:

 ```cmake [<options>] -S <path-to-source> -B <path-to-build>```

Reply:
Actually that didn't fix it for me..

Reply:
Yeah it looks like the `-B` flag was added in 3.13: https://cmake.org/cmake/help/v3.13/release/3.13.html

Strange the older version silently ignore it :)

Anyway I think it easy enough to avoid the `-B` flag rather then downloading custom cmake?  What do you think?

Reply:
I'll convert the makefile to `cd` to the build directory rather than use `-B`. Its disappointing that cmake chose to deploy a new flag that old versions ignore, though :(